client/src/components/UserPage.tsx

```diff
@@ -24,34 +26,34 @@ const UserPage = ({ match }: RouteComponentProps<RouteInfo>) => {
   const [isIdentityChecked, setIsIdentityChecked] = useState(
     user.identity_check
   );
-
   const { getAccountsByUser, accountsByUser } = useAccounts();
   const { usersById, getUserById } = useUsers();
   const { itemsByUser, getItemsByUser } = useItems();
   const userId = Number(match.params.userId);
 
   // functions to check username and email against data from identity/get
-  const checkUserName = useCallback(
-    (names: string[]) => {
-      const usernameArray = user.username.split(' ');
-      // if both the first name and last name of the username in this app are included somewhere in the
-      // financial institution's names array, return true
-      return usernameArray.every(username => {
-        return names.some(identName => {
-          return identName.indexOf(username) > -1;
-        });
+  const checkUserName = useCallback((names: string[], fullname: string) => {
+    // in case user enters "Last name, First name"
+    fullname = fullname.replace(',', ' ');
+    const usernameArray = fullname.split(' ');
```

Comment:
I think we should rename `username` here and in the rest of this function with something else that indicates this is a fullname that is split into substrings. Maybe `fullnameArray` or `fullnameTokens` or `fullnameSplitStrings`